includes/Admin/Admin_Page.php

```diff
@@ -64,13 +67,18 @@ public function add_hooks() {
 	/**
 	 * Adds the admin page under the tools menu.
 	 *
+	 * @since n.e.x.t Added multisite support.
 	 * @since 1.0.0
 	 */
 	public function add_page() {
-		$this->hook_suffix = add_management_page(
+		$admin_parent_page = is_multisite() ? 'settings.php' : 'tools.php';
+		$capabilities      = is_multisite() ? 'manage_network_plugins' : 'activate_plugins';
```

Comment:
This variable should probably be called `$capability` since it only stores _one_ capability.